wodles/aws/aws-s3.py

```diff
@@ -714,6 +776,112 @@ def iter_regions_and_accounts(self, account_id, regions):
         self.db_maintenance('', self.bucket)
 
 
+class AWSService(WazuhIntegration):
+    """
+    Class for getting AWS Inspector logs
+    :param access_key: AWS access key id
+    :param secret_key: AWS secret access key
+    :param profile: AWS profile
+    :param iam_role_arn: IAM Role
+    :param only_logs_after: Date after which obtain logs.
+    :param region: Region of service
+    """
+    def __init__(self, access_key, secret_key, aws_profile, iam_role_arn,
+        service_name, only_logs_after, region):
+
+        WazuhIntegration.__init__(self, access_key=access_key, secret_key=secret_key,
+            aws_profile=aws_profile, iam_role_arn=iam_role_arn,
+            service_name=service_name, region=region)
+
+        self.only_logs_after = only_logs_after
+        self.region = region
+
+    def format_message(self, msg):
+        return {'integration': 'aws', 'aws': msg}
```

Comment:
Is this a custom behaviour for a service? I think this could be common for both services and buckets.

Reply:
Yes, I checked that format is more complex for buckets (see get_alert_msg or get_alert_msg methods from AWSLogsBucket class). AWS Service (Inspector, by the moment) only needs this format to be sent to the socket.